src/S3/MultipartCopy.php

```diff
@@ -132,20 +133,28 @@ private function createPart($partNumber, $partsCount)
         }
         // The source parameter here is usually a string, but can be overloaded as an array
         // if the key contains a '?' character to specify where the query parameters start
+        $bucket = '';
+        $key = '';
         if (is_array($this->source)) {
             $key = str_replace('%2F', '/', rawurlencode($this->source['source_key']));
-            $data['CopySource'] = '/' . $this->source['source_bucket'] . '/' . $key;
+            $bucket = $this->source['source_bucket'];
         } else {
-
             list($bucket, $key) = explode('/', ltrim($this->source, '/'), 2);
-            $data['CopySource'] = '/' . $bucket . '/' . implode(
+            $key = implode(
             '/',
             array_map(
                 'urlencode',
                 explode('/', rawurldecode($key))
             )
         );
         }
+
+        $data['CopySource'] = '/';
```

Comment:
We can set this to something else for now- maybe $uri or something like that?  I'm wondering if that would look cleaner than starting the copy source up here.